src/hints/execution.rs

```diff
@@ -1871,4 +1887,29 @@ mod tests {
             get_integer_from_var_name(vars::ids::STATE_ENTRY, &mut vm, &ids_data, &ap_tracking).unwrap().into_owned();
         assert_eq!(state_entry, Felt252::from(123));
     }
+
+    #[test]
+    fn test_set_fp_plus_4_to_tx_nonce() {
+        let mut vm = VirtualMachine::new(false);
+        vm.add_memory_segment();
+        vm.add_memory_segment();
+        vm.set_fp(3);
```

Comment:
As you do not insert any variable, you can just leave FP to 0.

Reply:
Good point. fixed in 0c72f85